### PR TITLE
Bump version to 3.1.0 and update gradle version

### PR DESCRIPTION
**Pull request by lydiasama**
https://github.com/nimblehq/android-templates/projects/3#card-64963181

## What happened 👀

- Bump version accordingly to 3.1.0
- Update Gradle version

### CODE REVIEW

File: RxJavaTemplate/build.gradle [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext {
-        build_gradle_version = '4.1.1'
+        build_gradle_version = '4.2.0'
 
         android_version_code = 9
-        android_version_name = '0.7.0'
+        android_version_name = "3.1.0"
```
Comment (luongvo) [inline RxJavaTemplate/build.gradle:8]: 👍

**Review by luongvo — APPROVED** [review]

**Review by minhnimble — APPROVED** [review]
LGTM 🚀

File: RxJavaTemplate/build.gradle [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext {
-        build_gradle_version = '4.1.1'
+        build_gradle_version = '4.2.0'
```
Comment (chornerman) [inline RxJavaTemplate/build.gradle:5]: Should we use `4.2.2` instead?

Comment (lydiasama) [reply RxJavaTemplate/build.gradle:5]: Let me update the latest version. 😄

**Comment by lydiasama** [comment]
@chornerman Updated.

**Review by chornerman — APPROVED** [review]
LGTM krub 🎉

**Pull request merged**